src/components/RefineSearch/CancelSubmitButtonGroup.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, Icon, ButtonGroup } from "@nypl/design-system-react-components"
+import type { SyntheticEvent } from "react"
+
+interface CancelSubmitButtonGroupProps {
+  cancelHandler: (e: SyntheticEvent) => void
+  submitLabel: string
+  cancelLabel: string
+  formName: string
+  disableSubmit?: boolean
+}
+
+const CancelSubmitButtonGroup = ({
+  cancelHandler,
+  cancelLabel,
+  submitLabel,
+  formName,
+  disableSubmit,
+}: CancelSubmitButtonGroupProps) => {
+  const submitIcon = submitLabel.toLowerCase().includes("search")
```

Comment:
The use cases are straightforward and it makes sense to have this check here. I think it makes this brittle because it depends on the label to have "search". Thinking from the DS pov, have the consumer pass in the icon name that they want to render. This is a suggestion but up to you if you want to update this.

Reply:
I was on the fence about this, but I'm hedging my bets that this pattern will actually hold.